### PR TITLE
Let the caller organize the shutdown procedure.

### PR DESCRIPTION
**Pull request by michaeldejong**
This PR removes the automatic shutdown procedure. Library users are required to deregister their services themselves.

### CODE REVIEW

File: src/main/java/me/magnet/consultant/Consultant.java [hunk 1]
```diff
@@ -281,11 +281,6 @@ public Consultant build() {
 					http, pullConfig);
 
 			consultant.init(properties);
-
-			if (!executorSpecified) {
```
Comment (AlexKolpa) [inline src/main/java/me/magnet/consultant/Consultant.java]: Seems like `executorSpecified` is now unused?

Comment (michaeldejong) [inline src/main/java/me/magnet/consultant/Consultant.java]: Yup, you're right. Not sure why intelliJ didn't catch it. I'll remove it.

Comment (michaeldejong) [inline src/main/java/me/magnet/consultant/Consultant.java]: https://github.com/Magnetme/consultant/commit/2f8c3304773b6f4c5c4ba89a4f3ce03af98379b0

Comment (AlexKolpa) [inline src/main/java/me/magnet/consultant/Consultant.java]: :+1:

**Pull request merged**